perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -1384,6 +1384,39 @@ public static List<RichUser> getRichAdmins(PerunSession sess, PerunBean compleme
 		return AuthzResolverBlImpl.getRichAdmins(sess, complementaryObject, specificAttributes, role, onlyDirectAdmins, allUserAttributes);
 	}
 
+	/**
+	 * Get all valid user administrators (for group-based rights, status must be VALID for both Vo and group) for complementary object and role.
+	 *
+	 * If <b>onlyDirectAdmins</b> is <b>true</b>, return only direct users of the complementary object for role.
+	 *
+	 * @param sess perun session
+	 * @param complementaryObject for which we will get administrator
+	 * @param role expected role to filter managers by
+	 * @param onlyDirectAdmins if true, get only direct user administrators (if false, get both direct and indirect)
+	 *
+	 * @return list of user administrators for complementary object and role.
+	 */
+	public static List<User> getAdmins(PerunSession sess, PerunBean complementaryObject, String role, boolean onlyDirectAdmins) throws PrivilegeException, RoleCannotBeManagedException {
```

Comment:
Did you consider unifying this with `getRichAdmins`,since it does the same thing as `getRichAdmins` with the `specificAttributes` parameter empty? Since RichUser extends User I don't think it would cause problems.

Reply:
I considered it, and it seemed like a better option for me to add one new method than recast the output of the existing one. Yeah, `RichUser` is just an extension of `User`, but at the moment, we don't know where and how are these old methods used, so I wanted to be sure that it won't cause problems.